src/mapeo-project.js

```diff
@@ -625,6 +631,15 @@ export class MapeoProject extends TypedEmitter {
     return deviceInfo.update(existingDoc.versionId, doc)
   }
 
+  /** @param {boolean} isArchiveDevice */
+  async [kSetIsArchiveDevice](isArchiveDevice) {
+    this.#isArchiveDevice = isArchiveDevice
+  }
+
+  get [kIsArchiveDevice]() {
```

Comment:
Nit: I know the return type is trivial, but I'd love if we could add add a return type:

```suggestion
  /** @returns {boolean} */
  get [kIsArchiveDevice]() {
```